dss/index/es/validator.py

```diff
@@ -48,33 +48,36 @@ def request_json(url):
     extra_fields = []
     extra_documents = []
     for document in index_data.keys():
-        schema_info = SchemaInfo.from_json(index_data[document])
-        if schema_info is not None:
-            try:
-                schema = request_json(schema_info.url)
-            except Exception as ex:
-                extra_documents.append(document)
-                logger.warning(f"Unable to retrieve JSON schema information from {document} in bundle {bundle_id} "
-                               f"because retrieving {schema_info.url} caused exception: {ex}.")
+        for doc_list_ind, document_content in enumerate(index_data[document]):
```

Comment:
Not quite sure I understand the intent of this hunk. BTW: if this is change is orthogonal to the general intent of the PR, it would make sense to commit it separately. Don't know if that is the case here, though.

Reply:
This logic is required for `scrub_index_data` to work with the new bundle format.